stockflux-securities-master/src/components/AddSecurityButton.css

```diff
@@ -0,0 +1,16 @@
+.add-security-button > button {
+    border: solid 1px var(--text-teal);
+    background: none;
+    font-family: var(--font-roboto-sans-serif);
+    color: var(--text-teal-light);
+    font-weight: var(--font-weight-bolder);
+    border-radius: 25px;
+    height: 50px;
+    width: 250px;
+    font-size: var(--font-xxxlarge);
+    text-align: center;
+  }
+
+.add-security-button > button:hover {
+  color: var(--text-teal);
+  }
```

Comment:
Very minor, but could you unindent this brace?

Reply:
yep done